crates/mpt/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "mpt"
+edition = "2021"
+version.workspace = true
+rust-version.workspace = true
+license.workspace = true
+
+[lints]
+workspace = true
+
+[dependencies]
+alloy-trie = { version = "0.8.0", default-features = false }
+alloy-rlp = { version = "0.3.8", features = ["arrayvec"] }
+alloy-primitives = { version = "1.3", features = ["map"] }
+
+arrayvec = "0.7"
+bincode = { version = "1.3", optional = true }
+rkyv = { version = "0.8", optional = true }
+serde = { version = "1.0", optional = true }
+
+[features]
+default = []
+rkyv = ["dep:rkyv"]
+serde = [
+    "dep:serde",
+    "dep:bincode",
+    "alloy-primitives/serde",
+    "alloy-trie/serde",
+]
+rlp_serialize = []
+orphan = []
```

Comment:
Should we keep these features, or aim to have a minimal take?
If that's the case it will delete a decent amount of code in some modules.